gemini/use-cases/retrieval-augmented-generation/rag_vector_embedding_in_bigquery.ipynb

```diff
@@ -0,0 +1,794 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    EMBEDDING_MODEL_ID = "llm_gecko"  # @param {type:"string"}
</pre><p>Can you clarify in the comments if these are unique per project? I'm not sure if these would need to be the specific model names for textembedding-gecko, etc?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>These are remote model names within bigquery dataset. So they are unique per dataset per project. I have added Embedding endpoint type and LLM end point type to clarify that these are unique model names to use.</p>